runtime.md

```diff
@@ -21,6 +21,7 @@ The value MAY be one of:
 
     Additional values MAY be defined by the runtime, however, they MUST be used to represent new runtime states not defined above.
 * **`pid`**: (int) is the ID of the container process, as seen by the host.
+The pid MAY be excluded or set to a value indicating empty or null if it is not seen by the host. 
```

Comment:
nit: there's some [trailing whitespace][1] on the current line.

[1]: https://travis-ci.org/opencontainers/runtime-spec/jobs/193148351#L293